### DISCUSSION TITLE
Are these Streams the same?  If not what is the functional difference?

### DISCUSSION THREAD

**Discussion by hagak** [blakeblackshear/frigate#7783] [Ask A Question] [answered]
I see in the docs examples of both ways but not sure what the functional difference is?  I find the second one easier to read.

```
garage:
      - "ffmpeg:http://192.168.30.40/flv?port=1935&app=bcs&stream=channel0_ext.bcs&user={FRIGATE_CAMERA_USER}&password={FRIGATE_CAMERA_PASSWORD}#video=copy#audio=copy#audio=opus"
```

```
garage:
      - http://192.168.30.40/flv?port=1935&app=bcs&stream=channel0_ext.bcs&user={FRIGATE_CAMERA_USER}&password={FRIGATE_CAMERA_PASSWORD}
      - "ffmpeg:garage#video=copy#audio=copy#audio=opus"
```

**Comment by NickM-27** [comment, marked as answer]
You are referring to go2rtc. The first option uses ffmpeg to handle ALL of the streaming data, meanwhile the second option has go2rtc handle the main video & audio. The reason the former is recommended is because reolink cameras have issues in their http-flv implementation that go2rtc doesn't handle but ffmpeg is able to handle.
